.travis.yml

```diff
@@ -19,6 +19,16 @@ install:
 script:
   - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then travis_wait 60 make; fi
 
+deploy:
+  skip_cleanup: true
+  provider: releases
+  api_key:
+    secure: "DlCyRM85KiUN6p9BvmBjf7gtSHfjbuiJhjIOP19drQfIwxDNvgdQXOV8zSOIS2g8bQK8LOUkUWbVlxRkA5RBVfNb1v8MUaRhByOJ8zl8JMSLs4XcUvcLFENp7Au/6HCWMrPg170f3NvoaJFBCyVFh8nYRys7PcRenRafnvEapTLA6gNMaS6T4PIenwwu/FFfE+jAmW3PHDx+Ln0sgmTpyV+/8MyJrS0X39HUeM9HgX5C0ErUMptlb4LIrN5feGeHSE9agOKT7yIGoa4UzXpm0WaYIhg291S417l8uCS8uCa4j2D7EUJJmYTKi7I56hCJmMqBWttHrQhSRLfe6PKiBEw+ieR8lhcv/B5MMuWO04oypRko1/iDub8rYclzcty7WMqWDp+Fl0wT7SIy9fHL9kE8moIItTQDVxRUV3YaLx2pxD4o8OHea8FFnH/I6AMZkikw0Gk1bZlUmYsH0vFghs45rY71tMoVLgk9WAqeIbcgTgq1XEtdokY+PbnZ01EdlQKLj4LA8Wf1KfCqId4EDycgSF/3TAcWiuPr7cJFV9zgyOI4dzr1lRkLJi9huK4MmTbA5NYsA6Ci/Ca1+1hAUfDwcbSjRTeM4s4sCQnxihzQo66Ba0F18kFGVadkrnh/g1DnsW6NeTyfuukU4JvDg4/4Cyqa0IoG2rdERWgo4Es="
```

Comment:
Will this work with my PAT? for anuvu/zot? it works for my fork.
https://docs.travis-ci.com/user/deployment/releases/#authenticating-with-an-oauth-token